addon/adapters/moderator.js

```diff
@@ -0,0 +1,26 @@
+import OsfAdapter from './osf-adapter';
+
+export default OsfAdapter.extend({
+    pathForType() {
+        return 'preprint_providers';
+    },
+    buildURL(modelName, id, snapshot, requestType, query) {
+        let provider = '';
+        if (query) {
+            provider = query.provider;
```

Comment:
A few of these could be destructured assignment I think
```javascript
{ provider } = query;
```

Reply:
Destructuring didn't seem to work :/

`{ provider } = query;` caused a syntax error and `let { provider } = query;` resulted in provider not being set. Did I miss something?